### ISSUE TITLE
Screen reader accessibility

### ISSUE THREAD

**Issue by leevi978** [Balve-user/garmeres_v3#31]

**Comment by mashazyu** [comment]
@leevi978 

I'd be interested in working on the topic of accessibility, if it's still available.
And I have a couple of questions. As far as I understand there are several different aspects of accessibility - aria lables, usage of semantic HTML, keyboard accessibility, usage of alternative text (https://bejamas.io/hub/guides/next-js-and-accessibility). 
- I checke a11 report and it looks like aria labels are in place (please correct me if I'm wrong),
- are there any specific areas of accessibility topic that you had in mind, when you were creating this ticket? Or it's a part of the ticket to verify that all accessibility areas are covered and screen reader works correctly?

Thank you in advance!

Kind regards,
Maria

**Comment by leevi978** [comment]
@mashazyu This issue is still available, and we at Garmeres would all be very grateful if you wanted to give it a shot.

Like I have mentioned before, so far there hasn't been anyone else working on this project, so I have been slacking with the issue descriptions as I assumed no one but me was reading them. So I apologize for that.

This issue is about testing the website out with a screen reader tool, and making sure that it is easy and logical to use. I am no expert in universal accessibility, so I haven't done a proper deep dive yet. But we do have someone in Garmeres who might be able to do a user test on a preview, if available.

One thing to note, however - A speech synthesis now exists for the northern sámi language (Davvisámegiella), and I would like for the northern sámi translation to be usable with a northern sámi screen reader tool. Hence, there are two caveats:
1. If labels or descriptions are missing in English, they are likely also missing in Davvisámegiella. You can ask me for specific labels, and I will do my best to translate them
2. The website needs to be tested with the northern sámi screen reader tool. If you don't speak the language, then you can put this off until later, and make a follow-up issue for it.

**Comment by leevi978** [comment]
And if you do some testing and realize that there are large amounts of work to be done - feel free to create issues for them!
